### PR TITLE
Feature mylisting

### PR DESCRIPTION
**Pull request by lakuleavadhutglobant**
My listing component is sub component of profile component

### CODE REVIEW

File: Application/Client/src/app/createProfile/component/createProfile.component.ts [hunk 1]
```diff
@@ -1,35 +1,59 @@
...
+      this.getProfileData(this.userEmail);
+    });
+
+  }
+
+  getProfileData(userMail){
+    this.userDetails = this.GetUserProfileUrl+this.userEmail;
+    this._cservice.observableGetHttp(this.userDetails ,null,false)
+      .subscribe((res:Response)=> {
+          this.userProfileData = res;
```
Comment (satishshewaleglobant) [inline Application/Client/src/app/createProfile/component/createProfile.component.ts]: We can use this single variable instead declaring other varible

File: Application/Client/src/app/createProfile/tpls/createProfile.html [hunk 2]
```diff
@@ -3,89 +3,170 @@
     <ol class="breadcrumb col-lg-11">
       <li><a>Home</a></li>
       <li><a>my profile</a></li>
-      <li class="active">edit info</li>
+      <li class="active">{{userProfileData.UserName}}</li>
```
Comment (satishshewaleglobant) [inline Application/Client/src/app/createProfile/tpls/createProfile.html]: UserName should be userName

File: Application/Client/src/app/createProfile/tpls/createProfile.html [hunk 3]
```diff
@@ -3,89 +3,170 @@
...
+      </div>
     </div>
   </div>
 </div>
-</div>
+
+
+
+
+<!--<div class="container-fluid" >-->
```
Comment (satishshewaleglobant) [inline Application/Client/src/app/createProfile/tpls/createProfile.html]: Remove this commented code

File: Application/Client/src/app/createProfile/component/createProfile.component.ts [hunk 4]
```diff
@@ -1,35 +1,59 @@
...
-    ngOnInit() {
-        this.getProfileData();
-    }
-
-    getProfileData(){
-
-    }
-
-
+  private GetUserProfileUrl = 'http://in-it0289/Userapi/api/user/GetUserProfile?userEmail=';
```
Comment (satishshewaleglobant) [inline Application/Client/src/app/createProfile/component/createProfile.component.ts]: This should come from settings.json

File: Application/Client/src/app/createProfile/tpls/createProfile.html (hunk 2 above)
Comment (lakuleavadhutglobant) [reply Application/Client/src/app/createProfile/tpls/createProfile.html]: back end team  is using PASACAL naming convention so we have used this variable as it is on front end side

File: Application/Client/src/app/createProfile/tpls/createProfile.html (hunk 3 above)
Comment (lakuleavadhutglobant) [reply Application/Client/src/app/createProfile/tpls/createProfile.html]: fixed

File: Application/Client/src/app/createProfile/component/createProfile.component.ts (hunk 4 above)
Comment (lakuleavadhutglobant) [reply Application/Client/src/app/createProfile/component/createProfile.component.ts]: for the configuration of api i am working on it

**Pull request merged**